rwkv/chat_with_bot.py

```diff
@@ -286,7 +226,7 @@ def load_thread_state(_thread: str) -> torch.Tensor:
             logits = load_thread_state('chat')
             new = f"{user}{separator} {msg}\n\n{bot}{separator}"
             # print(f'### add ###\n[{new}]')
-            logits = process_tokens(tokenizer.encode(new).ids)
+            logits = process_tokens(tokenizer.encode(new).ids, newline_adj=-999999999)
```

Comment:
Why the adjustment needed here?

Reply:
I'm not sure. I copy it from https://github.com/BlinkDL/ChatRWKV/blob/adbbdfb8015a2261375467b2103846e061614491/v2/chat.py#L333

Reply:
probably this means we shouldn't generate '\n as our first token